content/pages/employment/job-seekers/unemployment-support.md

```diff
@@ -70,7 +71,8 @@ Receiving unemployment will not affect your wages, but if you receive separation
 
 - Current resume or military job history
 
-If you’re unemployed and at risk of becoming homeless, call or visit your [local VA medical center](/facilities/) or Community Resource and Referral Center where VA staff are ready to help.
+If you’re unemployed and at risk of becoming homeless, call or visit your local VA medical center or Community Resource and Referral Center where VA staff are ready to help. <br>
```

Comment:
**Global Note/Question:** Re: the question of rural communities, I wonder if we should change instances of **local** VA medical center to **nearest** or **closest** VA medical center. Or, can we say **nearest VA health facility** since not all zip codes will return a medical center?